dist.sh

```diff
@@ -7,6 +7,7 @@ set -e
 
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
 mkdir -p $DIR/dist
+mkdir -p $DIR/dist/docker
```

Comment:
probably need to clear this directory first, too (for example if we ever removed a binary)